### DISCUSSION TITLE
Loading Segmentation

### DISCUSSION THREAD

**Discussion by fanjie01** [Abe404/root_painter#85] [Q&A]
Hi:
I encountered some problems when using rootpainter. There was a FileNotFoundError when I first ran it.
![image](https://user-images.githubusercontent.com/125536412/219308216-e6ed347b-aafd-4d6f-9ad4-66c770b718f3.png)
![image](https://user-images.githubusercontent.com/125536412/219308660-d8228828-06e0-4bd6-b2cd-c7ee88b13be9.png)
So I created an instructions folder in the sync directory. It seems to be working. But a new problem has emerged. Save and next can't appear. I don't know how to solve it. 
![image](https://user-images.githubusercontent.com/125536412/219311320-9e73d5b1-8046-467b-9fb6-41a4424c5458.png)
I would appreciate it very much if you could provide some help.

**Comment by Abe404** [comment]
Hi, Sorry, somehow I missed this message (I didn't see a notification).

It looks like the problem is that you have not started the server.
You need to start the server before starting the client.

To get started I recommend the colab tutorial. The tutorial is available here: https://colab.research.google.com/drive/104narYAvTBt-X4QEDrBSOZm_DRaAKHtA?usp=sharing

There's a video of me presenting RootPainter and then later doing the tutorial available here:
https://www.youtube.com/watch?v=73u73tBvRO4

Sorry for the delayed response.

Kind regards,
Abraham

**Reply by Abe404** [reply]
Just to clarify a bit more, the server will create the necessary folders for the client to function when it is first ran (including the instructions folder that was missing in this case).
